src/offspot_demo/maint-compose/Caddyfile.tmpl

```diff
@@ -0,0 +1,5 @@
+{{ fqdn }} {
```

Comment:
You can use environment variable directly instead of a template

Reply:
Done